pkg/helper/client.go

```diff
@@ -102,5 +113,13 @@ func ApplyDesiredState(nodeNetworkState *nmstatev1alpha1.NodeNetworkState) (stri
 	if len(desiredState) == 0 {
 		return "Ignoring empty desired state", nil
 	}
-	return set(string(nodeNetworkState.Spec.DesiredState))
+	outputSet, err := set(string(nodeNetworkState.Spec.DesiredState))
```

Comment:
you might be better off parsing the desired state yaml, iterating over desired bridges and the interfaces explicitly set inside them. please consider.

Reply:
Then maybe we want to reconsider this https://github.com/nmstate/kubernetes-nmstate/pull/140, when NM/nmstate is ready customers will configure vlan-filtering explicitly at k8s-nmstate, with the above PR we can provide that before having nmstate/NM

Reply:
In case we want to parse desiredState we all need to ignore bridges with "state: absent", would be nice to be able to ask nmstatectl about interfaces managed by it.

Reply:
After talking with nmstate as expected we are the owners so we should know what interface do we manage, so parsing desiredState have to be, since #140 maybe is too much work.

Reply:
I have add filtering by desiredState bridges so I am resolving this.